boards/launchxl/apps/blink

```diff
@@ -0,0 +1 @@
+../../../userland/examples/blink
```

Comment:
Why have a launchxl-specific apps directory?

Reply:
When we created the launchxl board directory we copied the structure of the others, such as nrf52dk (which has an apps directory as well).

This pattern seems kind of ambiguous, and we should probably get rid of it on all boards -- or decide to keep it.

Reply:
I don't think these board specific apps like this are useful. I'd remove it here and we can work on the other platforms later.

Reply:
Sounds good, will do